cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -601,9 +602,248 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
+          "CREATE ROLE deniedBoostedProcedureExecutor3",
+          "CREATE ROLE deniedBoostedProcedureExecutor4",
+          "CREATE ROLE adminProcedureExecutor",
+          "CREATE ROLE procedureGlobbing1",
+          "CREATE ROLE procedureGlobbing2",
+          "CREATE ROLE procedureGlobbing3",
+          "CREATE ROLE procedureGlobbing4",
+          "CREATE ROLE procedureGlobbing5"
+        )
+        p("The dbms privileges for procedure execution are assignable using Cypher administrative commands. They can be granted, denied and revoked like other privileges.")
+        p("include::dbms/execute-procedure-syntax.asciidoc[]")
+
+        section("The `EXECUTE PROCEDURE` privilege", "execute-procedure-subsection", "enterprise-edition") {
+          p(
+            """The ability to execute a procedure can be granted via the `EXECUTE PROCEDURE` privilege.
+              |A user with this privilege is allowed to execute the procedures matched by the <<procedure-name-globbing, name-globbing>>.
+              |The following query shows an example of how to grant this privilege:""".stripMargin)
+          query("GRANT EXECUTE PROCEDURE db.schema.* ON DBMS TO procedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureExecutor' can then run any procedure in the `db.schema` namespace. The procedure will be run using the users own privileges.")
+            statsOnlyResultTable()
+          }
+          p("The resulting role should have privileges that only allow executing procedures in the `db.schema` namespace:")
+          query("SHOW ROLE procedureExecutor PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'procedureExecutor'")
+            resultTable()
+          }
+
+          p(
+            """If we want to allow executing all but a few procedures, we can grant `EXECUTE PROCEDURES *` and deny the unwanted procedures.
+              |For example, the following queries allows for executing all procedures except `dbms.killTransaction` and `dbms.killTransactions`:""".stripMargin)
+          query("GRANT EXECUTE PROCEDURE * ON DBMS TO deniedProcedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          query("DENY EXECUTE PROCEDURE dbms.killTransaction? ON DBMS TO deniedProcedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          p("The resulting role should have privileges that only allow executing all procedures except `dbms.killTransaction` and `dbms.killTransactions`:")
+          query("SHOW ROLE deniedProcedureExecutor PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'deniedProcedureExecutor'")
+            resultTable()
+          }
```

Comment:
Not really correct `?` is matching exactly one character (also confirm that its correct in other places `?` are used)